core/server/data/schema/default-settings.json

```diff
@@ -118,5 +118,16 @@
         "public_hash": {
             "defaultValue": null
         }
+    },
+    "members": {
+        "members_public_key": {
```

Comment:
Tiny design detail:

This breaks the pattern of the settings key names a bit. We don't repeat the name of the namespace in the key itself.

e.g. `blog.active_timezone` vs `members.members_public_key`

Reply:
The problem with this is that settings are actually flat, and the nesting is just for us humans - so we would have a conflict with the `session_secret` setting. And then if we were to ever have a need for a public_key or private_key setting in the future we would have to namespace it :/ 

As an alpha product i didn't think we should take up valuable non-namespaced settings names!

Reply:
The settings service should probably be changed to `settingsCache.get('core.settings_secret')` 🤣 
Okay let's leave it like this for now 👍

Reply:
:D agreed!